### DISCUSSION TITLE
[Feat] Mushroom Resource / NFT

### DISCUSSION THREAD

**Discussion by Clarkro** [sunflower-land/sunflower-land#1337] [Art]
![M1 1](https://user-images.githubusercontent.com/111134731/184373166-216cccb7-daa3-40e5-b5ee-d340389dc38e.jpg)

I would like to discuss implementing my Mushroom resource into the game. When I brought this up to Steve he thought it was a great idea so I've been putting a great deal of effort into creating the assets needed. 

I would love and appreciate any suggestions or recommendations. 

Peace, Love, and Mushrooms 
<3

**Comment by digital-farms** [comment]
what is the perspective of the mushroom? look at how the tree is tilted, its crown covers the top of the trunk, if you do the same mushroom will obviously fit better into the overall picture)

**Reply by Clarkro** [reply]
ok thanks for the feedback.
